web/src/components/organisms/Project/Content/ContentList/hooks.ts

```diff
@@ -334,6 +334,7 @@ export default () => {
   const handleSearchTerm = useCallback(
     (term?: string) => {
       searchParams.set("searchTerm", term ?? "");
+      searchParams.set("page", "1");
```

Comment:
Fixed page returns to page 1 after searching on a page other than page 1 on main table.